.pipelines/e2e-aks-arc.yaml

```diff
@@ -8,15 +13,35 @@ jobs:
     - template: templates/install-oras.yaml
     - template: templates/install-bats.yaml
     - template: templates/package-helm-chart.yaml
-    - template: templates/publish-helm-chart.yaml
+    - ${{ if eq(parameters.releaseE2E, true) }}:
+      - template: templates/publish-helm-chart.yaml
+        parameters:
+          releaseTag: true
+    - ${{ if eq(parameters.releaseE2E, false) }}:
+      - template: templates/publish-helm-chart.yaml
+    - template: templates/checkout-upstream-repo.yaml
     - template: templates/aks-setup.yaml
     - template: templates/enable-arc.yaml
-    - template: templates/add-arc-extension.yaml
+    - ${{ if eq(parameters.releaseE2E, true) }}:
```

Comment:
Can you separate this part (lines 26-37) out into a separate file? For readability